src/components/button/bl-button.stories.mdx

```diff
@@ -189,6 +205,22 @@ If button has a limited width and a long text that can not fit in a single line,
   </Story>
 </Canvas>
 
+## Loading Buttons
+
+Custom loading text can be set with `loading-label` attribute.
+
+<Canvas columns={1}>
+  <Story name="Loading Button With Icon" args={{ loading: true, loadingLabel: 'Logging in...', content: 'Login', icon: 'account' }}>
+    {LoadingStateTemplate.bind({})}
+  </Story>
+  <Story name="Loading State Buttons" args={{ loading: true, content: 'Login' }}>
+    {SingleButtonTemplate.bind({})}
+  </Story>
+  <Story name="Loading Button Custom Label" args={{ loading: true, loadingLabel: 'Logging in...', content: 'Login' }}>
+    {SingleButtonTemplate.bind({})}
+  </Story>
+</Canvas>
```

Comment:
Can you add a few more examples:

* An icon only button, without label.
* `small` and `large` examples.
* `disabled` example. You may consider to merge `disabled` with one of above examples, since we don't expect any visual change on `disabled` state.

These are for preventing us to break any of those states accidentally.